pkg/usage/usage.go

```diff
@@ -206,11 +206,25 @@ func (u *Usage) SetVolumeCapacity(volCapG string) *Usage {
 	return u
 }
 
+// Wrapper for setting replica count for volume events
+// NOTE: This doesn't get the replica count in a volume de-provision event.
+// TODO: Pick the current value of replica-count from the CAS-engine
+func (u *Usage) SetReplicaCount(count, method string) *Usage {
+	if method == "volume-provision" && count == "" {
+		// Case: When volume-provision the replica count isn't specified
+		// it is set to three by default by the m-apiserver
+		u.SetAction("replica:3")
+	}
+	u.SetAction("replica:" + count)
```

Comment:
should there be an else here?

Reply:
Not unless we want to send something other than `replica:` for a `volume-deprovision` event.
Our limitation here is that we don't get the replica-count on volume-deprovision.

Reply:
Okay, I get the issue now an else is required.

Reply:
Fixed in `35d26d`